propositions_and_proofs.md

```diff
@@ -13,10 +13,11 @@ One strategy for proving assertions about objects defined in the
 language of dependent type theory is to layer an assertion language
 and a proof language on top of the definition language. But there is
 no reason to multiply languages in this way: dependent type theory is
-flexible and expressive, and there is no reason we cannot represent
+flexible and expressive, and it turns out that there is no reason we cannot represent
 assertions and proofs in the same general framework.
 
-For example, we could introduce a new type, ``Prop``, to represent
+As an example of layering languages on top,
```

Comment:
Now I see that this is an example of using the same general framework; the "could/would" formulation below makes it a bit confusing. Saying, more directly, "... it turns out that we can represent assertions and proofs in the same general framework.        For example..."  , would help.